libnetwork/drivers/bridge/setup_ip_tables.go

```diff
@@ -429,3 +430,15 @@ func clearEndpointConnections(nlh *netlink.Handle, ep *bridgeEndpoint) {
 	}
 	iptables.DeleteConntrackEntries(nlh, ipv4List, ipv6List)
 }
+
+func clearConnectionsToUDPPorts(nlh *netlink.Handle, ep *bridgeEndpoint) {
```

Comment:
I wonder if this should be folded into `clearEndpointConnections`. Should UDP connections also be flushed when revoking external connectivity? (i.e. the other caller of `clearEndpointConnections`)

Reply:
> Should UDP connections also be flushed when revoking external connectivity? (i.e. the other caller of clearEndpointConnections)

From my testings, conntrack entries are deleted when the listening UDP socket got closed. If at this time, UDP packets still flow in, we're back to the bug reported in #44688. In the other hand, if UDP packets stop coming when the socket is closed, there's no more conntrack entries to worry about.

However, clearing conntrack entries as a precautionary measure won't hurt, so I'm ok doing this.